mep/common/views.py

```diff
@@ -43,6 +47,56 @@ def dispatch(self, request, *args, **kwargs):
         return response
 
 
+class RdfViewMixin(ContextMixin):
+    '''View mixin to add an RDF linked data graph to context for use in serializing
+    and embedding structured data in templates.'''
+
+    #: default schema.org type for a View
+    rdf_type = SCHEMA_ORG.WebPage
+    #: breadcrumbs, used to render breadcrumb navigation. they should be a list
+    #: of tuples like ('Title', '/url')
+    breadcrumbs = []
+    #: json-ld context for the rdf graph; defaults to schema.org
+    json_ld_context = str(SCHEMA_ORG)
+
+    def get_context_data(self, *args, **kwargs):
+        '''Add generated breadcrumbs and an RDF graph to the view context.'''
+        context = super().get_context_data(*args, **kwargs)
+        context['rdf'] = self.as_rdf().serialize(format='json-ld', auto_compact=True,
+                                                 context=self.json_ld_context, indent=4)
+        context['breadcrumbs'] = self.get_breadcrumbs()
+        return context
+
+    def get_uri(self):
```

Comment:
I wonder if `get_absolute_url` would be a better name, for consistency with Django objects? This kind of serves two purposes, from the RDF perspective it's the URI for the view, so I think either works.

Reply:
agree; I'll change it